website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -73,6 +73,35 @@ export default EntryPoint;
 | `render`                         | `func`             |            (*)           | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized. It's **recommended to use the `children`** prop to benefit from a simpler setup.                                                                                                                                                                                                                                  |
 | `apolloClient`         | `ApolloClient`           |                       | -       | An optional instance of [ApolloClient](https://www.apollographql.com/docs/react/) to be used instead of the default one. This is usually the case when you need to configure the Apollo cache. See [`createApolloClient`](#createapolloclient).                                                                                                                                                                                                                                        |
 
+
+## ApplicationPageTitle
+
+Provides Custom Applications the ability to display a more useful (Human readable) page title in the browser tab based on the page location. The following format is used:
+
+```
+<Entry-point-uri-path> - <project-key> - Merchant Center
+```
+
+You can optionally add more contextual information to the default page title by rendering the `<ApplicationPageTitle>` component and passing additional values.
+The values are prepended to the default page title and concatenated with a `-` separator. If each additional value exceeds 24 characters length, it will be truncated in the middle.
+
+The application shell now has an instance of the `<ApplicationPageTitle>` component. Page titles in the browser tabs will now automatically get more readable titles.
+
+This feature is available from version `21.14.0` onwards.
+
+Overwriting the default page title is recommended in detail pages where there is a human-readable resource identifier, for example a product name.
+
+
+```js
+import { ApplicationPageTitle } from '@commercetools-frontend/application-shell';
+
+<ApplicationPageTitle additionalParts={['T-Shirt red']} />;
+// T-Shirt red - Products - my-shop - Merchant Center
+```
+
+The `<ApplicationPageTitle>` component can be rendered multiple times and the last one rendered will overwrite the other ones.
+
+
```

Comment:
We should also document the "guidelines" for determining when to override a page title and what is the expected outcome. There is some info about it in the RFC, in particular the mapping table. Having a table here would also help, maybe a reduced version of it from an (external) Custom Application perspective.

Reply:
Thank you.
Feedback updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/44d19f71300483477867849422e6096f345e9ed4